static/versionDetails.json

```diff
@@ -155,7 +155,7 @@
     },
     {
       "flag": "isBlockStorageEnabled",
-      "description": "Use block devices instead of the filesystem for storage in the Ceph cluster.",
+      "description": "Use block devices instead of the filesystem for storage in the Ceph cluster. This flag will be automatically set for versions 1.4.3+.",
```

Comment:
```suggestion
      "description": "Use block devices instead of the filesystem for storage in the Ceph cluster. This flag will automatically be set to true for version 1.4.3+ because block storage must be enabled for these versions.",
```
This better explains the rationale, which also clarifies that this cannot be overridden by setting it to false—it's just always true.